### PR TITLE
Add Rails 5 to Appraisals

### PR DESCRIPTION
**Pull request by glaucocustodio**

### CODE REVIEW

File: gemfiles/rack_1.5.2.gemfile [hunk 1]
```diff
@@ -1,33 +1,33 @@
 # This file was generated by Appraisal
 
-source 'https://rubygems.org'
+source "https://rubygems.org"
 
-gem 'rack', '1.5.2'
+gem "rack", "1.5.2"
```
Comment (dblock) [inline gemfiles/rack_1.5.2.gemfile]: You don't like single quotes? :) Run `rubocop -a`.

**Comment by dblock** [comment]
There're some legit build failures! Glad we're doing this.

**Comment by dblock** [comment]
Similar issue: https://github.com/rails/rails/issues/1730 and http://stackoverflow.com/questions/5985136/ruby-json-parse-returns-undefined-method-bytesize-for, haven't looked at how to fix this.

File: gemfiles/rack_1.5.2.gemfile (hunk 1 above)
Comment (glaucocustodio) [inline gemfiles/rack_1.5.2.gemfile]: These replaces were automatic, I didn't do it manually.

Comment (dblock) [inline gemfiles/rack_1.5.2.gemfile]: Yep, Appraisal does that, then rerun `rubocop -a` to clean this up.

Comment (glaucocustodio) [inline gemfiles/rack_1.5.2.gemfile]: `rubocop -a` does not correct it.

Comment (dblock) [inline gemfiles/rack_1.5.2.gemfile]: You're right, these need to be added to the rubocop.yml like so:

screen shot 2016-07-26 at 11 26 05 am

Comment (glaucocustodio) [inline gemfiles/rack_1.5.2.gemfile]: Done.

**Comment by dblock** [comment]
The broken build is fixed in https://github.com/ruby-grape/grape/pull/1457, would you please rebase/squash? Thanks and sorry about that.

**Comment by dblock** [comment]
I've also simplified the build matrix by a lot in https://github.com/ruby-grape/grape/pull/1458.

File: gemfiles/rack_1.5.2.gemfile (hunk 1 above)
Comment (glaucocustodio) [inline gemfiles/rack_1.5.2.gemfile]: Adding `gemfiles/*` to `rubocop.yml` auto correct the `"` problem, however rubocop will compain about the syntax of gemfiles.

File: .travis.yml [hunk 2]
```diff
@@ -10,6 +10,8 @@ matrix:
         - bundle exec danger
     - rvm: 2.3.1
       gemfile: gemfiles/rack_1.5.2.gemfile
+    - rvm: 2.3.1
+      gemfile: gemfiles/rack_5.gemfile
```
Comment (dblock) [inline .travis.yml]: Typo: `rails_5.gemfile`.

**Comment by dblock** [comment]
Replaced this PR with https://github.com/ruby-grape/grape/pull/1464.

**Pull request closed without merging**